phoenix-core/src/main/java/org/apache/phoenix/jdbc/PhoenixDatabaseMetaData.java

```diff
@@ -224,6 +224,7 @@
     public static final String TASK_PRIORITY = "TASK_PRIORITY";
     public static final String TASK_DATA = "TASK_DATA";
     public static final String TASK_TABLE_TTL = "864000";
+    public static final int TTL_FOR_MUTEX = 15 * 60; // 15min
```

Comment:
just wondering, why 15 mins instead of 10 mins or 20 mins.

Reply:
I haven't changed the value of the TTL. It was 15 min originally

Reply:
We just picked a long enough time so that the client that created the mutex would be able to complete the metadata operation.